config/config.go

```diff
@@ -37,10 +38,21 @@ func InitConfig(project, region, bucket, table, awsProfile, owner, awsProviderVe
 
 // FindConfig loads a config and its version into memory
 func FindConfig(fs afero.Fs, configFile string) ([]byte, int, error) {
+	switch configFile {
+	case "fogg.json":
```

Comment:
does this depend on users not using the command line flag to change the default config?